homeassistant/components/cloud/__init__.py

```diff
@@ -187,10 +187,11 @@ def is_cloudhook_request(request):
             await cloud.remote.disconnect()
             await prefs.async_update(remote_enabled=False)
 
-    hass.services.async_register(
-        DOMAIN, SERVICE_REMOTE_CONNECT, _service_handler)
-    hass.services.async_register(
-        DOMAIN, SERVICE_REMOTE_DISCONNECT, _service_handler)
+    empty_schema = vol.Schema({})
```

Comment:
Is it worth moving this to cv? I feel like there are a lot of services that should be providing an empty_schema but are not.

Reply:
Or the async_register_admin_service create an own empty schema if the option is not provided

Reply:
oh I like that last one.